.env

```diff
@@ -0,0 +1,25 @@
+# Django Secret Key
+APP_SECRET_KEY = 'django-insecure-mxd#*-dutjdhw5_4-39*0nfz00#5cuntlr*uuqhktvqjurdt9_'
```

Comment:
This is the secret for the application on the Docker, we'll use it as an example, so this could be anything except the real secret key used by the application.